{{ cookiecutter.repo_name }}/LICENSE

```diff
@@ -1,8 +1,21 @@
-The MIT License (MIT)
-Copyright (c) 2016 DrivenData, Inc.
+MIT License
 
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+Copyright (c) 2024 fema-ffrd
```

Comment:
Is this an actual/legit copyright? If so, wouldn't it just be FEMA, and not fema-ffrd?